### ISSUE TITLE
Submit solution information

### ISSUE THREAD

**Issue by erbase** [exercism/cli#1004]
I'm trying to submit solution via the CLI tool, but I always receive this answer: "The exercise you are submitting doesn't have the necessary metadata", so my question is this: which file should I submit?
Many thanks for your answer

Eros

**Comment by SleeplessByte** [comment]
Hi there @erbase. Which track are you trying to submit for?

**Comment by glennj** [comment]
I hit this today: I was trying to submit the solution file, and a `./my_lib/my_custom_lib` file. My workaround was to move "my_custom_lib" into the same directory as the solution file. @erbase, does this sound familiar?

**Comment by SleeplessByte** [comment]
@glennj which track :) ? The file will need to be inside the `config.json` `files` array for it to submit, or we potentially have a bug.

**Comment by glennj** [comment]
I can upload arbitrary files that are in the current directory in the bash track: example https://exercism.org/tracks/bash/exercises/grains/solutions/glennj

If I try to _edit_ then online, I appropriately can't:
<img width="1057" alt="exercism edit unknown file" src="https://user-images.githubusercontent.com/122470/132412880-180150a9-57b9-4e0f-9391-11b29ebf1ecf.png">

Same on the Tcl track: https://exercism.org/tracks/tcl/exercises/forth/solutions/glennj

**Comment by SleeplessByte** [comment]
@glennj yeah, that makes sense. If they are in the `config.json` `solution` array, they are editable.
